tests/data/test1086

```diff
@@ -17,7 +17,7 @@ timing-dependent
 # Overload some standard FTP responses to make them shorter and faster
```

Comment:
Now that you have added SLOWDOWN, this comment and the REPLY lines below that implement it can be removed to simplify the test case.

Reply:
will do